crates/context/src/lib.rs

```diff
@@ -323,22 +339,26 @@ impl ContextManager {
         }
 
         let handle = self.store.handle();
-
         let identity_key = ContextIdentityKey::new(context_id, invitee_id);
-
         if handle.has(&identity_key)? {
             return Ok(None);
         }
 
+        let proxy_contract = self.get_proxy_contract(context_id).await?;
         let context_exists = handle.has(&ContextMetaKey::new(context_id))?;
-
-        let mut config = (!context_exists).then(|| ContextConfigParams {
-            protocol: protocol.into(),
-            network_id: network_id.into(),
-            contract_id: contract_id.into(),
-            application_revision: 0,
-            members_revision: 0,
-        });
+        let mut config = if !context_exists {
+            Some(ContextConfigParams {
+                protocol: protocol.into(),
+                network_id: network_id.into(),
+                contract_id: contract_id.into(),
+                proxy_contract: proxy_contract.into(),
+                application_revision: 0,
+                members_revision: 0,
+            })
+        } else {
+            //should this be existing config?
```

Comment:
q

Reply:
if the context already exists, the config already exists as well.